### PR TITLE
XWIKI-19406: Layout alignment issue

### PR DESCRIPTION
**Pull request by Sereza7**
**Jira:** https://jira.xwiki.org/browse/XWIKI-19406

## PR Changes
* Removed the bootstrap grid from the card on the Help Home page
* Added style to the new card elements
* Added a couple classes to force the number of columns in a card list
* Updated structure on other pages of the help section.

## Video demo
https://up1.xwikisas.com/#XCvEXu2KSZCe4wDKI1s4-g

## Tests
Successfully passed `mvn clean install -Pquality,integration-tests,docker -f xwiki-platform-core/xwiki-platform-help/ -Dxwiki.test.ui.wcag=true`

## Note
Done during BFD on June the 8th

### CODE REVIEW

**Review by michitux — CHANGES_REQUESTED** [review]
Overall, this looks good and much cleaner. What I'm wondering is if this kind of "card" layout shouldn't be introduced more generically. The class name seems quite generic already, which might not be a good idea for such a specific thing to avoid collisions but which could be a good candidate for a proposal to introduce this as a generic UI element in XWiki. What do you think?

Apart from that, I think it would be nice to use less variables for the media query breakpoints. From my understanding of https://getbootstrap.com/docs/3.4/customize/#media-queries-breakpoints, these variables should already be available.

File: xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/Help/Code/VelocityMacros.xml [hunk 1]
```diff
@@ -402,6 +402,52 @@
...
+  width: 49%;
+}
+
+@media (max-width: 1200px) {
+  li.card {
+    width: 32%;
+  }
+}
+
+@media (max-width: 992px) {
```
[the remark this answers is not in the document]
Comment (Sereza7) [reply xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/Help/Code/VelocityMacros.xml]: Addressed in https://github.com/xwiki/xwiki-platform/pull/2206/commits/3e64fc331791a5c8b10236a953aaac2a62d9c7c4

**Comment by michitux** [comment]
> What I'm wondering is if this kind of "card" layout shouldn't be introduced more generically. The class name seems quite generic already, which might not be a good idea for such a specific thing to avoid collisions but which could be a good candidate for a proposal to introduce this as a generic UI element in XWiki. What do you think?

You didn't answer this question from what I can see.

**Comment by Sereza7** [comment]
Sorry TT

> What I'm wondering is if this kind of "card" layout shouldn't be introduced more generically. The class name seems quite generic already, which might not be a good idea for such a specific thing to avoid collisions but which could be a good candidate for a proposal to introduce this as a generic UI element in XWiki. What do you think?

As far as I could see, this layout is already used generically through all the help module. I think they look neat and could be really useful for some non technical pages. I have no idea where to start to propose it as a generic UI element in XWiki. Should we introduce it in a new module or extend the flamingo skin?

About the name, I chose this name since this made sense to me (and it was implied from the context and velocity macro names). However, from what I could see, when people usually talk about **cards**, they expect a precise structure *inside* the block. Here,  the card in itself is just how the block itself is displayed.

At last, so far the cards are used in very specific context, and proposing it as a generic UI would come with a bunch of bugs and expectations for this element from users.

**Pull request merged**